src/basketball/worker/core/GameSim.js

```diff
@@ -248,12 +248,22 @@ class GameSim {
      *
      */
     homeCourtAdvantage() {
+        var homeCourtModifier = g.homeCourtAdvantage / 100.0;
+        /*
+        if (homeCourtModifier > .1) {
+          homeCourtModifier = .1;
+        }
+        else if (homeCourtModifier < 0) {
+          homeCourtModifier = 0;
+        }
+        */
+
         for (let t = 0; t < 2; t++) {
             let factor;
             if (t === 0) {
-                factor = 1.01; // Bonus for home team
+                factor = 1 + homeCourtModifier; // Bonus for home team
```

Comment:
Do you think + or * makes more sense? Ultimately users can achieve the same result either way, but I think * might be less confusing. Not sure.

Reply:
Yeah I think you're probably right. And inversely on the away team we can do 1 / homeCourtModifier, to ensure that number never goes negative